x/did/keeper/msg_server_did_test.go

```diff
@@ -0,0 +1,71 @@
+package keeper
```

Comment:
If the package declares the same as the actual logic, an error due to the circular reference will occur.
So `TestSuite` is not available.
The test recommends another package (`xxx_test`).

Reply:
Ya. It's tricky to use the `keeper_test` package name here, because we're testing private stuff: `verifyDIDOwnership()`. 

Actually, this test file doesn't need to use the `testsuite`, since we don't need actual keepers in these tests. 

In the future, if we really need to use the `testsuite`, let's consider to define some interfaces as you said.

Reply:
Ah. Currently, the `x/did/handler_test.go` contains tests for `msgServer.CreateDID()` and so on. I think it's better to move those tests to this file: `msg_server_did_test.go`, because the `handler.go` is just an wrapper of `msg_server_did.go` from now on. Then, we need keepers for testing and we'll face the circular reference as you mentioned.
Let's me think.

Reply:
IMHO, it would be okay to change `verifyDIDOwnership()` to `VerifyDIDOwnership()`, so that you can use the `keeper_test` package name.

It'd be simpler than defining some interfaces (that might be over-engineering).

Reply:
OK, I'll moved to `msg_server_did_test`

umm.. I think it would be good to proceed in two ways.

1. Integration tests use `testsuite` (package : `keeper_test`)
2. private test use `suite` (package : `keeper` )

Reply:
Oh, I don't have to use the suite.

Reply:
Yeah. The test suite has nice concepts such as `BeforeTest` and `AfterTest`. But, if the test is super simple, I think it's okay to not use the test suite. The basic Go `testing` package would be enough.

Reply:
OK.